### ISSUE TITLE
praha.idnes.cz

### ISSUE THREAD

**Issue by mapx-** [uBlockOrigin/uAssets#859]
`https://praha.idnes.cz/byvala-nemocnice-jilove-u-prahy-prodej-dhx-/praha-zpravy.aspx?c=A171108_133001_praha-zpravy_mav`

background images are not blocked anymore
2-3 days ago was working (ublock filters) this filter: 
`idnes.cz##script:inject(abort-on-property-write.js, bmone2n)`

- chrome 63 / FF 57
- uBlock Origin version: 1.14.19.100 / 1.14.19rc0
- default setup

I tried to add (FF 57) this filter
`idnes.cz##script:inject(abort-current-inline-script.js, bmone2n)`
I get:
TypeError: can't redefine non-configurable property "bmone2n"

**Comment by gorhill** [comment]
Would you want commit rights to be able to add such filters yourself?

**Comment by mapx-** [comment]
In this case (above) I don't know what to do :)  See also the error in console (firefox).

Probably I could help (adding commits) in many cases but not always (probably I can find docs how-to commit a change / filter ?)

**Comment by gorhill** [comment]
> probably I can find docs how-to commit a change / filter ?

I will just give you commit right and from there learn at your own pace. If you are not sure about a specific filter then just ask for an opinion. 

The error message is not terribly important, it's just nice to avoid it if possible so as to make it more difficult for a web master to figure out how uBO does it job on their site. But it's ok to have an error if unavoidable, in the end the goal is to remove the nuisance.

**Comment by gorhill** [comment]
So I sent you an invitation for commit rights, closing issues, etc.

To be clear: there is zero expectation that you have to ever commit even once. It's all volunteer work, and I give you the commit right strictly to remove an obstacle if _ever_ you do feel like committing a filter yourself rather than wait on someone else.

**Comment by mapx-** [comment]
Sure, got it.

**Comment by okiehsch** [comment]
@mapx- if you have questions how to commit a filter you can use this new github [feature](https://github.com/blog/2471-introducing-team-discussions) unless @gorhill prefers such discussions to be in the relevant issue?

**Comment by mapx-** [comment]
ok, thanks.

**Comment by gorhill** [comment]
I was not aware of this feature, so I don't have an opinion really. If there is any question I will try to answer wherever it's posted.

**Comment by okiehsch** [comment]
@gorhill I can't reproduce this issue, but this site renders the picker unusable on my end.
If I select the picker it does nothing and I get this error message.
![unnt 2](https://user-images.githubusercontent.com/16838044/33185963-7dd63c48-d086-11e7-9c4d-10876f61a73b.JPG)

**Comment by mapx-** [comment]
Sometimes I cannot reproduce it (in FF) but if I open the console (F12 or ctrl-shift-J) I get it ..always.

**Comment by gorhill** [comment]
Ok, I will investigate, uBO's content script should not fail.

**Comment by okiehsch** [comment]
I can only reproduce with Chrome uBO-version 1.14.18+, the picker works with Firefox uBO-version 1.14.19rc0 on my end.

**Comment by gorhill** [comment]
Not sure, but I think it's one of those site which removes uBO's injected `style` tag. This breaks uBO, I haven't accounted for this in the refactored code.

**Comment by okiehsch** [comment]
@mapx- I can't reproduce even if I open the console.
![unffddfg](https://user-images.githubusercontent.com/16838044/33187294-211a718a-d091-11e7-8ebd-f4c082005057.JPG)

**Comment by gotitbro** [comment]
> So I sent you an invitation for commit rights, closing issues, etc.

That's great. @mapx- is probably the first one to respond to most issues.

This issue is related to https://github.com/uBlockOrigin/uAssets/issues/462. @gorhill Should we open a new issue for a website or search if an issue is already there?
